addons/website/static/src/scss/website.scss

```diff
@@ -361,6 +366,20 @@ table.table_desc tr td {
     border-radius: 0;
 }
 
+.o_full_screen_height {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    min-height: 100vh;
+}
+
+.o_half_screen_height {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    min-height: 50vh;
+}
```

Comment:
I would write it this way:

```scss
.o_full_screen_height {
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    min-height: 100vh;
}
.o_half_screen_height {
    @extend .o_full_screen_height;
    min-height: 50vh;
}
```

Also, what about using extend in blog to share the min-height code too? Just to enforce what you are saying in the commit message.